src/gui/owncloudgui.cpp

```diff
@@ -552,6 +558,11 @@ void ownCloudGui::setupContextMenu()
     }
 }
 
+void ownCloudGui::setupContextMenuIfVisible()
+{
+    if (!_contextMenu || _contextMenu->isVisible())
```

Comment:
Hm, I added this with intent and it does work: `setupContextMenu()` builds the context menu if necessary. But one might just as well argue that setting up the context menu when it's not being shown yet is pointless. And `_contextMenu &&` would be less confusing to read. I'll update accordingly.